inline_scan.sh

```diff
@@ -120,7 +121,7 @@ get_and_validate_analyzer_options() {
     while getopts ':k:s:a:d:f:i:m:R:v:CPVho' option; do
         case "${option}" in
             k  ) k_flag=true; SYSDIG_API_TOKEN="${OPTARG}";;
-            s  ) s_flag=true; SYSDIG_BASE_SCANNING_URL="${OPTARG%%}";;
+            s  ) s_flag=true; SYSDIG_BASE_SCANNING_URL="${OPTARG%%}";SYSDIG_BASE_SCANNING_API_URL="${SYSDIG_BASE_SCANNING_URL}";;
```

Comment:
I assume SYSDIG_BASE_SCANNING_API_URL will be used for API requests and SYSDIG_BASE_SCANNING_URL will be used for the UI links (specifically the link to the report that is printed at the end?).

For on-prem, shouldn't we add /secure to the base path?

Reply:
I see it is added when printing the link in line 545.

Reply:
/secure is added when the script produces the result page link. It works also in on-prem installation.

Reply:
yes @airadier the on-prem url modification is handled via `-o` flag